pom.xml

```diff
@@ -47,12 +47,12 @@
         <repository>
             <id>jboss-releases-repository</id>
             <name>JBoss Releases Repository</name>
-            <url>https://repository.jboss.org/nexus/service/local/staging/deploy/maven2/</url>
+            <url>https://repository.jboss.org/nexus/repository/wildfly-extras/</url>
```

Comment:
Not sure if this is still true. I found contraindicative information.

Could be also https://repository.jboss.org/nexus/repository/jbossqe-eap/

Reply:
Either `jbossqe-eap` with direct deployment or `wildfly-extras` with workflow from https://docs.google.com/document/d/1-6_vmFWzRVuMRwSuv_nOgG530qSDe6Ipc_eCERv6lrE/edit?pli=1&tab=t.0#heading=h.d5smy1wvihbk

Reply:
I pick `jbossqe-eap` because it is simpler and fit our needs.